src/services/monitorDutchAuctionRebalanceEligibility.ts

```diff
@@ -65,26 +63,41 @@ const tryCreateDutchAuction = async (leverageToken: LeverageToken, pricers: Pric
 
     const receipt = await publicClient.waitForTransactionReceipt({
       hash: tx,
+      timeout: CREATE_AUCTION_TIMEOUT,
     });
 
     const createAuctionEvent = parseEventLogs({
       abi: DutchAuctionRebalancerAbi,
       eventName: "AuctionCreated",
       logs: receipt.logs,
     })[0];
-    const { status } = createAuctionEvent.args;
 
-    console.log(
-      `Rebalancer.CreateAuction successful for LeverageToken ${leverageToken.address}, auction created. Transaction hash: ${tx}`
-    );
+    if (receipt.status === "reverted") {
+      console.error(`Rebalancer.CreateAuction reverted for LeverageToken ${leverageToken.address}, transaction hash: ${tx}. This likely ocurred because the auction was already created by another rebalancer.`);
+      await sendAlert(
+        `*Error creating DutchAuctionRebalance*\n• LeverageToken: \`${leverageToken.address}\`\n• Transaction Hash: \`${tx}\`\n• This likely ocurred because the auction was already created by another rebalancer.`,
+        LogLevel.INFO
+      );
+
+      // If receipt status is reverted, re-simulate to get the error, as the receipt does not contain the error.
+      // If the simulation succeeds, the another attempt of this `tryCreateDutchAuction` function will need to be made
+      // (we will not try to create the auction again in this if statement)
+      await rebalancerContract.simulate.createAuction([leverageToken.address]);
```

Comment:
the `catch` that this logic is within handles the specific contract errors like `AuctionAlreadyExists` that we would typically see here